Sources/ProjectDescription/PackageSettings.swift

```diff
@@ -45,25 +45,41 @@ public struct PackageSettings: Codable, Equatable, Sendable {
     /// Custom project configurations to be used for projects generated from SwiftPackageManager.
     public var projectOptions: [String: Project.Options]
 
+    /// A Boolean value that indicates whether the test targets of local swift packages are included in generated project.
+    ///
+    /// The default value is true.
```

Comment:
@fortmarek recommended to make the new behavior a default

Reply:
For the default value, it was commented in https://github.com/tuist/tuist/pull/6436#discussion_r1658427312
Is there any other discussion that I've missed?

Reply:
I understand what a valid state is. 🙏